From f1ec141d606b5241fc410fa184f1f3efe3b48b87 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Tue, 2 May 2017 20:13:18 +0800
Subject: [PATCH] PM / devfreq: rockchip_dmc: set polling_ms to 50

In order to scaling frequency more timely, reduce the polling_ms.

Change-Id: Icbee5552396fa0552fb514d92ea77687228c3e28
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 drivers/devfreq/rockchip_dmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/devfreq/rockchip_dmc.c b/drivers/devfreq/rockchip_dmc.c
index 938be121e11e..dce8b4d945c1 100644
--- a/drivers/devfreq/rockchip_dmc.c
+++ b/drivers/devfreq/rockchip_dmc.c
@@ -220,7 +220,7 @@ static int rockchip_dmcfreq_get_cur_freq(struct device *dev,
 }
 
 static struct devfreq_dev_profile rockchip_devfreq_dmc_profile = {
-	.polling_ms	= 200,
+	.polling_ms	= 50,
 	.target		= rockchip_dmcfreq_target,
 	.get_dev_status	= rockchip_dmcfreq_get_dev_status,
 	.get_cur_freq	= rockchip_dmcfreq_get_cur_freq,
-- 
2.35.3

